src/components/contentSection/SearchManager.jsx

```diff
@@ -1,19 +1,19 @@
 import React, { useContext, useEffect, useState, useCallback } from "react";
 import shortid from "shortid";
 
-import { fetchTracks } from "./utils/spotify";
-import { setTracks, getTracks } from "./utils/tracksStorage";
-import parseArtistsTracks from "./utils/parser";
-import UserContext from "./contexts/UserContext";
-import SearchForm from "./SearchForm";
-import ResponsesContainer from "./ResponsesContainer";
-import CreatePlaylistForm from "./CreatePlaylistForm";
+import { fetchTracks } from "../../utils/spotify";
+import { setTracks, getTracks } from "../../utils/tracksStorage";
+import parseArtistsTracks from "../../utils/parser";
+import UserContext from "../../contexts/UserContext";
+import SearchForm from "./searchSection/SearchForm";
+import ResponsesContainer from "./searchResultsSection/ResponsesContainer";
```

Comment:
- 'Section' postfix can be removed. /search-results/SearchForm is much better.

- Also CamelCase notation usually used for jsx components (ie ResponsesContainer.jsx), but all other js files and folders usually used all-lowercase with '-' delimiters. For example, searchResults -> search-results etc